Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml

```diff
@@ -54,6 +54,7 @@
   parent: BaseToolSurgery
   description: A surgical tool used to make incisions into flesh.
   components:
+  - type: Sharp
```

Comment:
maybe add `butcherDelayModifier` here? scalpels are small tools, after all

Reply:
I agree, will look into this!